source/install/prod-rhel-6.rst

```diff
@@ -3,357 +3,4 @@
-     ssl_session_timeout 5m;
-     ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
-     ssl_ciphers 'EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH';
-     ssl_prefer_server_ciphers on;
-     ssl_session_cache shared:SSL:10m;
-
-     location / {
-        gzip off;
-        proxy_set_header X-Forwarded-Ssl on;
-        client_max_body_size 50M;
-        proxy_set_header Upgrade $http_upgrade;
-        proxy_set_header Connection "upgrade";
-        proxy_set_header Host $http_host;
-        proxy_set_header X-Real-IP $remote_addr;
-        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-        proxy_set_header X-Forwarded-Proto $scheme;
-        proxy_set_header X-Frame-Options SAMEORIGIN;
-        proxy_pass http://10.10.10.2:8065;
-     }
-  }
-
-6. Be sure to restart nginx
-  * ``\ sudo service nginx start``
-7. Add the following line to cron so the cert will renew every month
-  * ``crontab -e``
-  * ``@monthly /home/YOURUSERNAME/letsencrypt/letsencrypt-auto certonly --reinstall -d yourdomainname && sudo service nginx reload``
-8. Check that your SSL certificate is set up correctly
-  * Test the SSL certificate by visiting a site such as `https://www.ssllabs.com/ssltest/index.html <https://www.ssllabs.com/ssltest/index.html>`_
-  * If there’s an error about the missing chain or certificate path, there is likely an intermediate certificate missing that needs to be included
-
-Test setup and configure Mattermost Server
-------------------------------------------
-
-1. Navigate to ``https://mattermost.example.com`` and create a team and
-   user.
-2. The first user in the system is automatically granted the
-   ``system_admin`` role, which gives you access to the System Console.
-3. From the ``town-square`` channel click the dropdown and choose the
-   ``System Console`` option
-4.  Update **Notification** > **Email** settings to setup an SMTP email service. The example below assumes AmazonSES.
-
-   -  Set *Send Email Notifications* to ``true``
-   -  Set *Require Email Verification* to ``true``
-   -  Set *Feedback Name* to ``No-Reply``
-   -  Set *Feedback Email* to ``mattermost@example.com``
-   -  Set *SMTP Username* to ``[YOUR_SMTP_USERNAME]``
-   -  Set *SMTP Password* to ``[YOUR_SMTP_PASSWORD]``
-   -  Set *SMTP Server* to ``email-smtp.us-east-1.amazonaws.com``
-   -  Set *SMTP Port* to ``465``
-   -  Set *Connection Security* to ``TLS``
-   -  Save the Settings
-
-5. Update **File** > **Storage** settings:
-
-   -  Change *Local Directory Location* from ``./data/`` to
-      ``/opt/mattermost/data``
-
-6. Update **General** > **Logging** settings:
-
-   -  Set *Log to The Console* to ``false``
-
-7. Update **Advanced** > **Rate Limiting** settings:
-
-   -  Set *Vary By Remote Address* to ``false``
-   -  Set *Vary By HTTP Header* to ``X-Real-IP``
-
-8. Feel free to modify other settings
-9. Restart the Mattermost Service by typing:
-
-   -  ``sudo restart mattermost``
+This document has been superceded by :doc:`install-rhel-66`.
```

Comment:
superceded > superseded. Also, should we be using the active form? ie "This document is superseded by..."?